lib/puma/const.rb

```diff
@@ -239,5 +239,11 @@ module Const
     # Mininum interval to checks worker health
     WORKER_CHECK_INTERVAL = 5
 
+    # Illegal character in the key or value of response header
+    ILLEGAL_HEADER_KEY_REGEX = /[\u0000-\u0025|#{Regexp.escape('"(),/:;<=>?@[]{}')}]/.freeze
```

Comment:
maybe `Regexp.escape('"(),/:;<=>?@[]{}')` can be extracted to improve readibility?
```ruby
ILLEGAL_HEADER_CHAR_REGEX = Regexp.escape('"(),/:;<=>?@[]{}').freeze
ILLEGAL_HEADER_KEY_REGEX = /[\u0000-\u0025|#{ILLEGAL_HEADER_CHAR_REGEX}]/.freeze
```

Reply:
I agree, maybe add another constant for it 👍